src/cider/nrepl/middleware/debug.clj

```diff
@@ -435,14 +445,13 @@
       (try
         (read-string code)
         (catch Exception e)))
-    (binding [*skip-breaks* (atom nil)]
```

Comment:
Fair point. I guess I considered this to be somewhat related, in that I made sure to carefully examine all uses of `*skip-breaks*`, and saw that this one was a leftover. Happy to move it to a separate PR if you prefer.